.github/workflows/ci.yml

```diff
@@ -1,78 +1,95 @@
-        path: src/${{ github.event.repository.name }}
-        fetch-depth: 0
-    - name: Move source code
-      run: mv $GITHUB_WORKSPACE/src /opt/carma/
-    - name: Checkout dependencies
-      run: |
-        ./src/${{ github.event.repository.name }}/carma-messenger-core/docker/checkout.bash -r /opt/carma/
-    - name: Build Ros1
-      run: |
-        source "$INIT_ENV"
-        PACKAGES=$(find . -maxdepth 2 -type f -name package.xml | sed 's/\.\///' | cut -d/ -f1)
-        sed -i '/colcon build/ s/$/ --parallel-workers 4 --packages-up-to $PACKAGES/' /home/carma/.ci-image/engineering_tools/code_coverage/make_with_coverage.bash
-        make_with_coverage.bash -m -e /opt/carma/ -o ./coverage_reports/gcov
-    - name: Run Ros1 C++ Tests
-      continue-on-error: true
-      run: |
-        source "$INIT_ENV"
-        sed -i '/colcon test/ s/$/ --parallel-workers 4 --packages-above $PACKAGES/' /home/carma/.ci-image/engineering_tools/code_coverage/make_with_coverage.bash
-        make_with_coverage.bash -t -e /opt/carma/ -o ./coverage_reports/gcov
-    - name: Backup ROS1 compile_commands.json
-      run: |
-        mv /opt/carma/build/compile_commands.json /opt/carma/compile_commands.ros1.json        
-    - name: Cleanup before ROS 2 build
-      # Clear the build and install folders before building ROS 2 
-      run: |
-        rm -rf /opt/carma/install
-        rm -rf /opt/carma/build                   
-    - name: Build Ros2
-      run: |
-        source "$INIT_ENV"
-        source "$ROS_2_ENV"
-        PACKAGES=$(find . -maxdepth 2 -type f -name package.xml | sed 's/\.\///' | cut -d/ -f1)
-        sed -i '/colcon build/ s/$/ --parallel-workers 4 --packages-up-to $PACKAGES/' /home/carma/.ci-image/engineering_tools/code_coverage/make_with_coverage.bash
-        make_with_coverage.bash -m -e /opt/carma/ -o ./coverage_reports/gcov
-    - name: Run Ros2 C++ Tests
-      continue-on-error: true
-      run: |
-        source "$INIT_ENV"
-        source "$ROS_2_ENV"
-        sed -i '/colcon test/ s/$/ --parallel-workers 4 --packages-above $PACKAGES/' /home/carma/.ci-image/engineering_tools/code_coverage/make_with_coverage.bash
-        make_with_coverage.bash -t -e /opt/carma/ -o ./coverage_reports/gcov    
-    - name: Combine ROS1 and ROS2 compile_commands.json files
-      continue-on-error: true
-      run: |
-        mv /opt/carma/build/compile_commands.json /opt/carma/compile_commands.ros2.json
-        jq -s add /opt/carma/compile_commands.ros1.json /opt/carma/compile_commands.ros2.json > /opt/carma/build/compile_commands.json             
-    - name: Run SonarScanner
-      uses: usdot-fhwa-stol/actions/sonar-scanner@main
-      with:
-        sonar-token: ${{ secrets.SONAR_TOKEN }}
-        working-dir: "/opt/carma/src/${{ github.event.repository.name }}"
+name: "CI: Run tests"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: 
+      - develop
+      - master
+      - "release_candidate/*"
```

Comment:
Is this naming convention for release candidates? Normally I see `rc-*` or something similar.

Reply:
Not it is not. This is something I had locally and didn't mean to commit